src/main/java/com/synopsys/integration/detect/lifecycle/autonomous/model/ScanSettings.java

```diff
@@ -0,0 +1,86 @@
+package com.synopsys.integration.detect.lifecycle.autonomous.model;
+
+import java.util.SortedMap;
+import java.util.SortedSet;
+import java.util.TreeMap;
+import java.util.TreeSet;
+import java.util.Optional;
+
+/**
+ * ScanSettings is a data model used to store the properties and scan targets used by Detect for the given or
+ * derived <code>detect.source.path</code> value, when Detect's autonomous scanning mode is enabled.
+ * <br>If Detect has access to a scan settings JSON file from a previous run on the <code>detect.source.path</code>,
```

Comment:
need for the html tags?

Reply:
No specific need. Just wanted to format the javadoc better - is there a way to achieve new lines without using html tags like <br>?